apps/build-onchain-apps/package.json

```diff
@@ -28,6 +28,7 @@
     "next": "13.5.6",
     "next-pwa": "5.6.0",
     "perfume.js": "9.2.0",
+    "pino-pretty": "^10.3.1",
```

Comment:
Hey Pino what's up.
What do we use this for?

Reply:
It's a transitive dep, but the build was failing since it was being required but wasn't in the node_modules. The package that requires it had a comment that we have to include it separately